charts/fluentd/templates/deployment.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "fluentd.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
i dont think these are needed, the resources will be created in the release's namespace automatically

Reply:
Thanks! Let me triple check. Will push everything when i can!